### ISSUE TITLE
TiledMap Collision

### ISSUE THREAD

**Issue by LokiMidgard** [craftworkgames/MonoGame.Extended#704]
I started working on a TiledMapCollision manager, since I haven't seen one in this repo.

I haven't pushed it to GitHub yet. I tried to use the already existing collision API, but I customized it for my project. This can probably be corrected, but I would need some feedback. If there is interest.

**Comment by J23t** [comment]
Are we talking polygons? 
I have a remade the _CreateGrid_ at _CollisionWorldExtensions_ in [my fork](https://github.com/J23t/MonoGame.Extended/commit/d992d497d2a25516eb5ee57a9154a722adfbcb69).
It seems to work but only for one rectangle per tile.

**Comment by LokiMidgard** [comment]
I have published the [code](https://github.com/LokiMidgard/monogame-playground/blob/development/SpaceSim/TileMapCollision.cs) in the meantime.

It is also based on the `CollisionWorldExtension`

It should work on multiple collision boxes per tile and collide with object layer. But it is not pollished yet. It uses Circles instead of ellipses and does not support rotation for anything.

No other geometry that is used by Tiled is supported yet. And it need to be testet more...

It also only does colliding and has no gravity (I was thinking on a top down tilemap not jump&run)

**Comment by janfokke** [comment]
The collision feature will probably be revisited all together. 
https://github.com/craftworkgames/MonoGame.Extended/issues/620#issuecomment-631240278

@lithiumtoast  started [experimenting ](https://github.com/lithiumtoast/Collision) with collision a while ago. I suggest you get in touch with him.

**Comment by lithiumtoast** [comment]
Thanks for opening the issue. Others are having questions about this is as well. I have knowledge on how to do this but there are some other things that need to be done first. I'll leave this open as a reminder.

**Comment by AristurtleDev** [comment]
Closing this issue as part of the tilemap system replacement.

The legacy `TiledMap` implementation has been deprecated and is being fully replaced by the new format-agnostic tilemap architecture tracked in #1113.

No additional feature work or non-critical fixes are being made to the legacy system. The new `Tilemap` + parser + renderer architecture resolves this class of issues by design.

If you are still using `TiledMap`, please plan to migrate to the new tilemap API before the next major release, where the legacy system will be removed.

If after migrating to the new system you encounter equivalent behavior, please open a new issue referencing #1113 with a minimal repro.

Thank you.
